### ISSUE TITLE
fit, evaluate model with Goodbooks

### ISSUE THREAD

**Issue by yemiajayi** [lyst/lightfm#386]
Beginner and using the [Quickstart](https://lyst.github.io/lightfm/docs/quickstart.html) to familiarize myself with the *lightfm*  project. I tried using the [Building datasets](https://github.com/lyst/lightfm/tree/master/examples/dataset) example to fit and evaluate the LightFM model. Wasn't sure how to create a training and test set as in the *Quickstart* example so I tried cross-validating as below. It sure wasn't right because of the output from Precision@k. Can someone please put me through?

**Building interaction matrix**
```...```

```python
from lightfm.cross_validation import random_train_test_split

train, test = random_train_test_split(interactions, test_percentage=0.3, 
                                      random_state=np.random.RandomState(43))

train_features, test_features = random_train_test_split(item_features, test_percentage=0.3, 
                                      random_state=np.random.RandomState(43))
```

```python
from lightfm import LightFM

model = LightFM(loss='warp')
model.fit(train, item_features=train_features, epochs=30, num_threads=4, verbose=False)
```
```
<lightfm.lightfm.LightFM at 0x248528c89e8>
```
```python
from lightfm.evaluation import precision_at_k

print("Train precision: %.2f" % precision_at_k(model, train, k=5, item_features=train_features).mean())
print("Test precision: %.2f" % precision_at_k(model, test, k=5, item_features=test_features).mean())
```
```
Train precision: 0.01
Test precision: 0.01
```

**Comment by DoronGi** [comment]
You do not need to split ```items_features``` to train and test. Split only interactions (and interaction weight if you are using it). ```item_features``` describes all the items, and doesn't contain interaction data.
